components/tidb_query_expr/src/impl_vec.rs

```diff
@@ -58,24 +58,6 @@ mod tests {
     use super::*;
     use crate::types::test_util::RpnFnScalarEvaluator;
 
-    fn real_eq(a: Real, b: Real) -> bool {
```

Comment:
Why? The original PR doesn't seem to have this part.

Reply:
This part is modified in prev-pr. I merge the #17330.

Reply:
Okay. I will review this PR after #17330.